remove refs to qa forum (#1974)
opensafely/documentation: docs/how-to-get-help.md

## Slack

You will be invited to join the Bennett Institute's workspace during the co-pilot programme.

> If you're outside the co-pilot programme,

Reviewer: ```suggestion
```

> then your second port of call should be to ask your question in any channel to which you have access.

Reviewer: ```suggestion
Please feel free to ask questions in any channel to which you have access.
```
